src/RefreshToolbar/RefreshToolbar.tsx

```diff
@@ -0,0 +1,102 @@
+import { FC, ReactNode, useMemo } from 'react';
+import Button from 'src/Button/Button';
```

Comment:
Minor: this could probably be moved to be near the theme import so default export imports are grouped.